### PR TITLE
Optimize whole setup with docker

### PR DESCRIPTION
**Pull request by wysow**
Hello,

First thanks for this docker version of the Broadway Demo Application.

Here are some updates:

- synchronized against master branch of original application
- optimized Dockerfile
- optimized startup script as the `server` namespace doesn't seems to be present (I don't really understand why at this time, advices welcome...)

### CODE REVIEW

**Comment by othillo** [comment]
@wysow do you mind rebasing your PR? I rebased mine already because it couldn't be merged.

**Comment by wysow** [comment]
@othillo just done, internal php server is now launch by dedicated Symfony console command.

**Comment by wysow** [comment]
@othillo do it seems right now or should I optimize something more?

File: composer.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
     },
     "autoload": {
         "psr-0": {
-            "": "src/"
+            "": ["src/", "test/"]
```
Comment (othillo) [inline composer.json]: I don't think this is necessary as the test directory is already loaded in autoload-dev

Comment (wysow) [inline composer.json]: Sure you're right, I have just optimized this and deleted a useless paragraph in `README.md`

**Comment by othillo** [comment]
thank you, @wysow !

**Pull request merged**